### ISSUE TITLE
train special classes

### ISSUE THREAD

**Issue by huixiancheng** [Halmstad-University/SalsaNext#57]
Could you give me some advices on how to train the two hard sem.seg classes: Motorcyclist and Other-ground.In my experiment, They are completely close to 0.
Notice that in the pre-trained model,there also nearly 0 , as follow.

>   train_update : 0.00010502038594856752
  train_loss : 0.35978143828302084
  train_acc : 0.9487481341277788
  train_iou : 0.7022054133173021
  valid_loss : 1.4024676781763712
  valid_acc : 0.9072582666167708
  valid_iou : 0.6136875937614279
  best_train_iou : 0.7011966601875725
  best_val_iou : 0.6177246216099593
  train_hetero : 0
  unlabeled :  0.,
  car :  0.9437
  bicycle :0.4768
  motorcycle : 0.5139
  truck : 0.8114
  other-vehicle : 0.4819
  person : 0.7155
  bicyclist : 0.8241
  motorcyclist : 0.
  road : 0.9459
  parking : 0.3692
  sidewalk : 0.8166
  other-ground : 0.0103
  building : 0.8843
  fence : 0.5629
  vegetation : 0.8482
  trunk : 0.6664
  terrain : 0.7131
  pole : 0.6047
  traffic-sign : 0.4711
  valid_heteros : 0
But in your paper,the results are much better.(**Motorcyclist:19.4 Other-ground:29.1**)
Could you share tyour experience?
![QQ截图20210412225837](https://user-images.githubusercontent.com/65054204/114415860-a8b0b800-9be2-11eb-85f9-6f502341a6b0.png)

**Comment by TiagoCortinhal** [comment]
Hello!
I assume the results you are showing are the mIoU for the validation and done on the projection maps? Not the LiDAR Point Clouds? 

The results on the paper are inferred for the test set (and for the entire point cloud), if you want to run for the test set you need to check SemanticKITTI webpage on how to do it (they have a codalab for that).

Best,
Tiago
